alpha/alpha-server/src/main/resources/application.yaml

```diff
@@ -65,3 +68,23 @@ spring:
     properties:
       eclipselink:
         ddl-generation: none
+
+---
+spring:
+  profiles: spring-cloud
+eureka:
+  client:
+    enabled: true
+    service-url:
+      defaultZone: http://0.0.0.0:8761/eureka
+  instance:
+    metadataMap:
+      servicecomb-alpha-server: ${server.host}:${alpha.server.port}
+    ip-address: ${server.host}
+    preferIpAddress: true
+    nonSecurePortEnabled: true
+    securePortEnabled: false
+    homePageUrl: http://${server.host}:${server.port}
+    healthCheckUrl: http://${server.host}:${server.port}/health
+    statusPageUrl: http://${server.host}:${server.port}/info
```

Comment:
It's spring-cloud 1.x address

Reply:
Non-required parameters are removed in application.yml